Automattic/wp-calypso
# Prepare the domain page to work with the experiment - backend strategy

#### Proposed Changes

* Prepare the domain page to work with the experiment. We're focusing on the New header, background color, and centralizing the content. Domain filtering section [will be handled in another task](https://github.com/Automattic/wp-calypso/issues/72894), as well as [the Domain Header](https://github.com/Automattic/wp-calypso/issues/72892).


<img width="1066" alt="image" src="https://user-images.githubusercontent.com/1044309/216463226-d4ae84f8-1a3b-417b-b54f-40ff949ee384.png">


#### Testing Instructions



* Apply this diff to your sandbox API: D100077-code
* Access the current Domain upsell flow starting at `Upgrade` button on your side menu
* The domain page should look like the production one (assuming you're at the control group)
* Change your variation to `treatment` at the experiment `21025-calypso-sidebar-upsell-dedicated-upgrade-flow-202301` following these tips: pb5gDS-2nD-p2#connecting-an-experiment-to-a-jitm
* **IMPORTANT**: First access [the Calypso home](http://calypso.localhost:3000/), and after click at the CTA on the left menu (to ensure our [CSS rules](https://github.com/Automattic/wp-calypso/pull/72932/files#diff-e9d5bbfeb0cc388a1ec2501f08f49524a97724605eca06bc1f3f626bf12cfab4R105) will be applied as expected)
* Make sure your change will work as expected
* You should now see the new interface, make sure it aligns with our design expectations: pebzTe-Fx-p2#comment-1131
* Make sure the `calypso_upgrade_nudge_cta_click` (with property cta_name: free-to-paid-sidebar) is the correct exposure event for triggering this experiment.

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/72889
